doc/guides/backport-to-lts-branches.md

```diff
@@ -0,0 +1,124 @@
+# How to backport to LTS branches
+
+### Set up your repo
+
+- Collaborator access
+
+- Make sure nobody else is currently backporting to the same LTS branch
+
+- Clone node checkout staging branch
+  ```bash
+  git clone git@github.com/nodejs/node --origin DANGER
+  ```
+
+- Make sure up to date with staging
+  ```bash
+  git fetch --all
+  git checkout -B v8.x-staging DANGER/v8.x-staging
+  ```
+
+- Run the tests to make sure everything works:
+  ```bash
+  ./configure && make -j4 test
+  ```
+
+- Install branch-diff (and set up the key)
+  ```bash
+  npm install -g branch-diff
+  ```
+
+### Triage PRs
+
+Go through [PRs with the v8.x label](https://github.com/nodejs/node/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Av8.x), for each PR:
+
+- Run CI
+- Check approvals (you can approve yourself)
+- Check SemVer
+- Check the PR metadata wasn't changed from `master` commit.
+- If there are merge conflicts, comment in the issue asking the PR author to rebase.
+- Land if okay
+  - Add git alias if you haven't already:
+  ```bash
+  git config --global alias.pa '!curl -L $1.patch | git am -3 -S --whitespace=fix #'
+  ```
+  - Patch Pull Request onto staging branch:
+  ```bash
+  git pa https://github.com/nodejs/node/pull/12345
+  ```
+  - If it doesn't apply cleanly then abort with:
+  ```bash
+  git am --abort
+  ```
+  - If it lands and tests pass, then add the `Backport-PR-URL:` line to each commit message:
+  ```bash
+  # If a single commit:
+  git commit --amend
+
+  # If multiple commits:
+  git rebase -i @~4 # If 3 commits.
+  # Then change pick to reword for each commit
+  #
+
+  # Add this line to each commit under the PR-URL:
+  # Backport-PR-URL: https://github.com/nodejs/node/pull/12345
+  ```
+  - Test the commit and push it
+    ```bash
+    ./configure && make -j4 test
+    git push
+    ```
+  - Close the backport PR with `Landed in $(git rev-parse --short HEAD)`, update the label on the original PR from `backport-requested-v8.x` to `backported-to-v8.x`.
+
+### Triage commits
+
+- Run branch diff /w parameters
+  ```bash
+  branch-diff ???
```

Comment:
branch-diff vX.x-staging upstream/vY.x -exclude-label semver-major,semver-minor,dont-land-on-vX.x,backport-requested-vX.x,backported-to-vX.x,baking-for-lts --filter-release

where X = LTS branch and Y = Current branch